crates/subspace-runtime/src/lib.rs

```diff
@@ -1018,6 +1018,7 @@ impl_runtime_apis! {
             match &ext.function {
                 RuntimeCall::Subspace(call) => Subspace::is_inherent(call),
                 RuntimeCall::Timestamp(call) => Timestamp::is_inherent(call),
+                RuntimeCall::Domains(call) => Domains::is_inherent(call),
```

Comment:
Would be nice to sort this alphabetically, also similar place in test runtime should probably be updated as well

Reply:
Changed